features/add_md_ref.feature

```diff
@@ -0,0 +1,109 @@
+@us40
```

Comment:
have amended as suggested, I just did that for quickness when typing the --tags etc